.idea/livekit-server.iml

```diff
@@ -4,8 +4,6 @@
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/bin" />
-      <excludeFolder url="file://$MODULE_DIR$/pkg/routing/routingfakes" />
-      <excludeFolder url="file://$MODULE_DIR$/pkg/service/servicefakes" />
```

Comment:
do we want to add a gitignore line for .idea?

Reply:
seeing changes here isn't normal, I had changed that earlier but found out it messes with the tests